config/events.go

```diff
@@ -0,0 +1,149 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	validator "github.com/asaskevich/govalidator"
+)
+
+// VASTEventElement indicates valid VAST event element
+type VASTEventElement string
+
+const (
+	ImpressionVASTElement             VASTEventElement = "impression"
+	TrackingVASTElement               VASTEventElement = "tracking"
+	ClickTrackingVASTElement          VASTEventElement = "clicktracking"
+	CompanionClickThroughVASTElement  VASTEventElement = "companionclickthrough"
+	ErrorVASTElement                  VASTEventElement = "error"
+	NonLinearClickTrackingVASTElement VASTEventElement = "nonlinearclicktracking"
+)
+
+var vastEventElementMap = map[VASTEventElement]struct{}{
+	ImpressionVASTElement:             {},
+	TrackingVASTElement:               {},
+	ClickTrackingVASTElement:          {},
+	CompanionClickThroughVASTElement:  {},
+	ErrorVASTElement:                  {},
+	NonLinearClickTrackingVASTElement: {},
+}
+
+// TrackingEventType indicates quartile events
+type TrackingEventType string
+
+const (
+	Start         TrackingEventType = "start"
+	FirstQuartile TrackingEventType = "firstQuartile"
+	MidPoint      TrackingEventType = "midPoint"
+	ThirdQuartile TrackingEventType = "thirdQuartile"
+	Complete      TrackingEventType = "complete"
```

Comment:
Are we missing some event types? I was having a hard time figuring out what the valid values are but I did notice in the [Macros](https://docs.google.com/document/d/1a-U1DytiCTtacM27qH3OZwhPa7bDwQzfpVRHDhyikWw/edit#heading=h.mgqqskl4hzs6) section of the PDR that there were more values listed than what we have here. I know this section is focused on macros but it is referring to tracking event types so I figured this was the list of valid values:
`Everything else is a direct translation of the tracking event types: start, firstQuartile, midpoint, thirdQuartile, complete, mute, unmute, pause, rewind, resume, fullscreen, exitFullscreen, expand, collapse, closeLinear, skip`
Thoughts?

Reply:
@bsardo : As per[ Phase I Document - Scope](https://docs.google.com/document/d/1zzi06jWpYsvhuiQ_3KCpes0DpmpO2teatmo4V5kIhi0/edit#heading=h.kdqvvrw0rbzm) we are scoping to release on `start, firstQuartile, midPoint, thirdQuartile, complete`
@bretg  : Please confirm

Reply:
@ShriprasadM thanks, I missed that. I was looking at a different document.